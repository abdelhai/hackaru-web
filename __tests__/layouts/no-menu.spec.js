import Factory from '@/__tests__/__setups__/factory';
import NoMenu from '@/layouts/no-menu';

describe('NoMenu', () => {
  it('render correctly', () => {
    const factory = new Factory(NoMenu, { stubs: ['nuxt'] });
    expect(factory.shallow().element).toMatchSnapshot();
  });
});
