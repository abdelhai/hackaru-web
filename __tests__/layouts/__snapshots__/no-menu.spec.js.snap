// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMenu render correctly 1`] = `
<div
  class="wrapper"
  ontouchstart=""
>
  <side-bar-stub />
   
  <div
    class="content"
  >
    <nuxt-stub />
  </div>
</div>
`;
